pipelines/basic.yaml

```diff
@@ -128,6 +128,10 @@ spec:
       type: string
       description: "Deploy timeout"
       default: "900"
+    - name: DEPLOY_OPTIONAL_DEPS_METHOD
```

Comment:
Consider including this option for `basic_no_iqe.yaml` as well

Reply:
Done :)